README.md

```diff
@@ -7,5 +7,6 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
-Aditya Narayan <narayanaditya95@gmail.com>  
+Abinash Meher <abinashdakshana999@gmail.com><br/>
```

Comment:
You should add a new empty line instead of adding `<br/>`, also correct it for Aditya's entry. Have a new line after every line with name.